packages/cli/src/commands/buildpacks/index.ts

```diff
@@ -22,11 +22,20 @@ export default class Index extends Command {
         Accept: 'application/vnd.heroku+json; version=3.sdk',
       },
     })
-    const buildpacks = await buildpacksCommand.fetch(flags.app, getGeneration(app) === 'fir')
+    const isFirApp = getGeneration(app) === 'fir'
+    const buildpacks = await buildpacksCommand.fetch(flags.app, isFirApp)
     if (buildpacks.length === 0) {
       this.log(`${color.app(flags.app)} has no Buildpacks.`)
     } else {
-      ux.styledHeader(`${color.app(flags.app)} Buildpack${buildpacks.length > 1 ? 's' : ''}`)
+      const pluralizedBuildpacks = buildpacks.length > 1 ? 'Buildpacks' : 'Buildpack'
+      let header = `${color.app(flags.app)}`
+      if (isFirApp) {
+        header += ` Cloud Native ${pluralizedBuildpacks} (from latest release OCI image)`
+      } else {
+        header += ` Classic ${pluralizedBuildpacks} (from buildpack registry)`
```

Comment:
```suggestion
        header += ` Classic ${pluralizedBuildpacks} (from the Heroku Buildpack Registry)`
```

Reply:
Adding Heroku to the header to distinguish it from the CNB buildpack registry (https://registry.buildpacks.io/)